doc/_templates/autosummary/class_no_inherited.rst

```diff
@@ -0,0 +1,90 @@
+{{ fullname | replace("pennylane", "qml") }}

```

Comment:
@trbromley I think this fixes it. Essentially, it provides an autosummary template specifically for the QNN classes, that ignores inherited methods. This is required because the `torch.nn.Module` and `tf.keras.Layer` classes include docstrings with invalid ReST :(

Reply:
Thanks @josh146, this is awesome